packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -216,11 +216,16 @@ export function getWebpackConfig(bundle: Bundle, worker: WorkerConfig) {
             loader: 'raw-loader',
           },
         },
+        {
+          test: /\.mjs$/,
+          include: /node_modules/,
+          type: 'javascript/auto',
+        },
       ],
     },
 
     resolve: {
-      extensions: ['.js', '.ts', '.tsx', '.json'],
+      extensions: ['.mjs', '.js', '.ts', '.tsx', '.json'],
```

Comment:
Why is this necessary?

Reply:
https://github.com/graphql/graphql-js/issues/1272